app/grants/views.py

```diff
@@ -208,7 +208,7 @@ def grant_details(request, grant_id, grant_slug):
         subscriptions = grant.subscriptions.filter(active=True, error=False).order_by('-created_on')
         cancelled_subscriptions = grant.subscriptions.filter(active=False, error=False).order_by('-created_on')
         _contributions = Contribution.objects.filter(subscription__in=grant.subscriptions.all())
-        phantom_funds = grant.phantom_funding.filter(round_number=3)
+        phantom_funds = grant.phantom_funding.filter(round_number=4)
```

Comment:
hmm i actually think this should be a `grant.phantom_funding.all()` not filtered by round number.. since we'll want to show the old phantom funding items on the grant details page too

Reply:
If we do that I think we'll need to update the grant details page as well to have two seperate sections and do the $5/funding selections separately

Reply:
why is that? i dont see anything on the grant details page (not grant/fund page) that denotes phantom funds seperately by round.

the phantom funds object here is just used to create the contributors + contributions list..

Reply:
All makes sense - thanks Kevin!